Jenkinsfile

```diff
@@ -78,9 +78,7 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                 checkout scm
                                 sh 'echo -n $(git rev-parse HEAD) > go/revision'
                                 sh "git add go/revision"
-                                env.GIT_COMMITTER_NAME = 'Jenkins'
-                                env.GIT_COMMITTER_EMAIL = 'ci@keybase.io'
-                                sh 'git commit --author="Jenkins <ci@keybase.io>" -am "revision file added"'
+                                sh "git stash"
```

Comment:
we read the file in the next line, so stashing here won't work. I'm suggesting to `stash` before diff_test and `stash pop` immediately after.